routes/userRoutes.js

```diff
@@ -7,13 +7,14 @@ const { User } = require('../models/User');
 const requireLogin = require('../middlewares/requireLogin');
 
 router.delete('/delete/:id', requireLogin, async (req, res) => {
-	try {
-		// Get the authenticated user's ID from req.user.id
-		const { id } = req.params;
+  try {
+    const { id } = req.params;
 
-		// Use Mongoose to find and delete the user by ID
-		console.log('Deleting user with ID:', id);
-		await User.findByIdAndDelete(id);
+    const deletedUser = await User.findByIdAndDelete(id);
+
+    if (!deletedUser) {
+      return res.status(404).json({ error: 'User not found' });
```

Comment:
We won't block the merge because of this, but we should have a look at the API guidelines i 
 general, here specifically: https://github.com/Educado-App/resources/blob/main/API_GUIDELINES.md#http-delete-returns-204